<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommonFields</class>
 <widget class="QWidget" name="CommonFields">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>275</width>
    <height>218</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CommonFields</string>
  </property>
  <widget class="QLabel" name="dateLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>150</y>
     <width>47</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Дата:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>80</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Диагноза:</string>
   </property>
  </widget>
  <widget class="DateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>150</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="wrapping">
    <bool>false</bool>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="currentSectionIndex">
    <number>0</number>
   </property>
  </widget>
  <widget class="SpinBox" name="priceSpinBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>180</y>
     <width>62</width>
     <height>21</height>
    </rect>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="prefix">
    <string/>
   </property>
   <property name="suffix">
    <string> лв.</string>
   </property>
   <property name="maximum">
    <double>10000.000000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>180</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Цена:</string>
   </property>
  </widget>
  <widget class="TextEdit" name="diagnosisEdit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="acceptRichText">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="TextEdit" name="manipulationEdit">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>20</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="acceptRichText">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Манипулация:</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DateEdit</class>
   <extends>QDateEdit</extends>
   <header>View/uiComponents/qt_derived/DateEdit.h</header>
  </customwidget>
  <customwidget>
   <class>TextEdit</class>
   <extends>QTextEdit</extends>
   <header>View/uiComponents/qt_derived/TextEdit.h</header>
  </customwidget>
  <customwidget>
   <class>SpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>View/uiComponents/qt_derived/SpinBox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
